sphinx_immaterial/apidoc/python/apigen.py

```diff
@@ -266,6 +268,10 @@ def _must_shorten():
         if not added_ellipsis:
             added_ellipsis = True
             ellipsis_node = sphinx.addnodes.desc_sig_punctuation("", "...")
+            # When using the `sphinx_immaterial.apidoc.format_signatures`
+            # extension, replace the text of this node to make it valid Python
+            # syntax.
+            ellipsis_node["munged_text_for_formatting"] = "___"
```

Comment:
Should probably merge #367 before reviewing this yes? It might be easier for me to differentiate what changes are actually specific to this PR's goal.

Reply:
This hunk is also in #367 --- I accidentally included it here originally, but I have now removed it.